workflow/envs/samtools_stats.yaml

```
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - samtools==1.10
```

our container has version 1.14, should we change to same here?
```suggestion
  - samtools==1.14
```

---

It is from the wrapper but I guess it should be ok to change